share/spice/near_earth_objects/near_earth_objects.js

```diff
@@ -0,0 +1,61 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_near_earth_objects = function(api_result) {
+
+        if (!api_result || !api_result.near_earth_objects || api_result.near_earth_objects.length === 0 || api_result.element_count === 0) {
+            return Spice.failed('near_earth_objects');
+        }
+        
+        // Put all NEOs into a single array
+        var neo_list = [];
+        for (var obj in api_result.near_earth_objects) {
+            if (api_result.near_earth_objects.hasOwnProperty(obj)) {
+                neo_list = neo_list.concat(api_result.near_earth_objects[obj]);
+            }
+        }
+        // Sort array in order of approach date
+        neo_list.sort(function(a, b) {
+            return parseInt(a.close_approach_data[0].epoch_date_close_approach) - parseInt(b.close_approach_data[0].epoch_date_close_approach);
+        });
```

Comment:
@tagawa You may be able to use the "sort_fields" option to accomplish this, same result in the end. 

http://docs.duckduckhack.com/frontend-reference/display-reference.html#sortfields-object-optional